### PR TITLE
Ingested TC 1978.

### PR DESCRIPTION
**Pull request by davidstap**
See https://aclanthology.org/www.mt-archive.info/70/Aslib-1978-TOC.htm for original.

### CODE REVIEW

**Comment by mjpost** [comment]
@mbollmann @akoehn What do you think about creating a `<backmatter>` tag to hold items like indices and so on? The motivating example here is [this paper (a list of participants)](https://aclanthology.org/www.mt-archive.info/70/Aslib-1978-participants.pdf), which I would like to ingest for its historical value.

**Review by mjpost — APPROVED** [review]
This looks good, except I'd like to add the list of participants. The simplest approach would be to add it as another paper with no authors, though I think it might be better to have a new tag for back matter, as noted.

File: data/xml/1978.tc.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      <year>1978</year>
+      <editor><first>Barbara M.</first><last>Snell</last></editor>
+    </meta>
+    <frontmatter>
+      <url hash="390aee7d">1978.tc-1.0</url>
+      <bibkey>tc-1978-translating</bibkey>
+    </frontmatter>
+    <paper id="1">
+      <title>Multilingual communication: Chairman’s introductory review of translating and the computer</title>
+      <author><first>J.C.</first><last>Sager</last></author>
```
Comment (mjpost) [inline data/xml/1978.tc.xml:18]: This is a bit of a pain, but I think we should open the PDFs and expand the names.

Comment (davidstap) [reply data/xml/1978.tc.xml]: Okay, I'll do that for the future editions. Thanks for doing this edition!

**Comment by mjpost** [comment]
It's unlikely everyone has time at this exact moment to resolve this. Let's do a workaround: add these as normal papers, but manually add a "type=backmatter" attribute to the `paper` tag. This will allow us to easily find them later, with minimal work now. You'll have to adjust the schema. Sound good?

e.g.,

`<paper id=74 type=backmatter>`

**Comment by davidstap** [comment]
Makes sense and sounds good, I will continue ASAP :-)

**Pull request merged**